Lib/test/test_fileinput.py

```diff
@@ -87,28 +87,35 @@ def close(self):
 class BufferSizesTests(unittest.TestCase):
     def test_buffer_sizes(self):
         # First, run the tests with default and teeny buffer size.
```

Comment:
The tests will be not ran with non-default buffer size. Please restore tests.

Reply:
Ok, but that is useless as `bufsize=` is anyway ignored. That is basically running the same exact test twice...